pkg/services/ngalert/api/api_ruler.go

```diff
@@ -234,7 +234,7 @@ func (srv RulerSrv) RouteGetRulesConfig(c *contextmodel.ReqContext) response.Res
 	}
 
 	dashboardUID := c.Query("dashboard_uid")
-	panelID, err := getPanelIDFromRequest(c.Req)
+	panelID, err := getPanelIDFromQuery(c.Req.URL.Query())
```

Comment:
Should this be `c.Req.Form`? I'm confused?

Reply:
True, I just tried to leave `api_ruler.go` as untouched as possible to keep the PR smaller.

I'll follow up in another PR.